mkdocs.yml

```diff
@@ -177,6 +177,7 @@ plugins:
         "development/generic-views.md": "development/core/generic-views.md"
         "development/getting-started.md": "development/core/getting-started.md"
         "development/homepage.md": "development/core/homepage.md"
+        "development/local-k8s.md": "development/core/local-k8s.md"
```

Comment:
Not really necessary as this is net-new documentation, not something that's been relocated and requires this redirect.

Reply:
Removed